src/NServiceBus.Core/Transports/Learning/LearningTransportInfrastructure.cs

```diff
@@ -59,7 +59,7 @@ string FindSolutionRoot()
                 if (parent == null)
                 {
                     // throw for now. if we discover there is an edge then we can fix it in a patch.
-                    throw new Exception("Couldn't find the solution directory for the learning transport.");
+                    throw new Exception("Couldn't find the solution directory for the learning transport. If you are running this endpoint outside the solution folder structure, make sure to specify a storage directory using the 'transportConfiguration.StorageDirectory' API.");
```

Comment:
Instead of `'transportConfiguration.StorageDirectory'`, this should be `'EndpointConfiguration.UseTransport<LearningTransport>().StorageDirectory()'`